pyro/distributions/util.py

```diff
@@ -22,7 +19,7 @@ def log_gamma(xx):
     x = xx - 1.0
     t = x + 5.5
     t = t - (x + 0.5) * torch.log(t)
-    ser = Variable(torch.ones(x.size()).type(ttype)) * magic1
+    ser = torch_ones_like(x) * magic1
```

Comment:
Nice simplification! Do you think we apply your earlier suggestion and simplify further to
```py
ser = magic1
for c in gamma_coeff:
    x = x + 1.0
    ser = ser + c / x
```
(I didn't write this code, I just maintain it :wink:)